ui/components/multichain/asset-picker-amount/asset-picker-modal/asset-picker-modal.tsx

```diff
@@ -260,6 +240,7 @@ export function AssetPickerModal({
                 name={t('tokens')}
                 tabKey="tokens"
               >
+                {/* FIXME: I don't think these are rendering */}
```

Comment:
not sure i understand the comment here , the code below will render the search input and the token list on the modal
<img width="370" alt="Screenshot 2024-02-28 at 12 10 37" src="https://github.com/MetaMask/metamask-extension/assets/26223211/fe7f4519-f865-4bfd-8b71-2cf2437b3ef9">

Reply:
Oops – good catch